test/chunking-form/index.js

```diff
@@ -62,11 +62,16 @@ describe('chunking form', () => {
 								expectedFiles = [];
 							}
 
-							assert.deepEqual(Object.keys(actualFiles), Object.keys(expectedFiles));
-
-							Object.keys(actualFiles).forEach(fileName => {
-								assert.strictEqual(actualFiles[fileName], expectedFiles[fileName], 'Unexpected output for ' + fileName);
-							});
+							(function recurse(actualFiles, expectedFiles, dirs) {
+								const fileNames = Array.from(new Set(Object.keys(actualFiles).concat(Object.keys(expectedFiles))));
+								fileNames.forEach(fileName => {
+									const sections = dirs.concat(fileName);
+									if (typeof actualFiles[fileName] === 'object' && typeof expectedFiles[fileName] === 'object') {
+										return recurse(actualFiles[fileName], expectedFiles[fileName], sections);
+									}
+									assert.strictEqual(actualFiles[fileName], expectedFiles[fileName], 'Unexpected output for ' + sections.join('/'));
+								});
+							})(actualFiles, expectedFiles, []);
```

Comment:
Any reason you are doing this as an IIFE and not just define a function variable and call it? Then you could even extract it as a helper out of this test and make the test more readable (provided you give it a better name than `recurse` 😉)